### PR TITLE
doxy and misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by nyalldawson** [comment]
@luzpaz cancelation is deliberate - we use the American spelling.

The others should also be added to the https://github.com/qgis/QGIS/blob/master/scripts/spell_check/spelling.dat file so that they are automatically caught by travis

File: src/app/dwg/libdxfrw/intern/drw_textcodec.cpp [hunk 1]
```diff
@@ -549,7 +549,7 @@ std::string DRW_Conv932Table::toUtf8( std::string *s )
 std::string DRW_ConvUTF16::fromUtf8( std::string *s )
 {
   DRW_UNUSED( s );
-  //RLZ: to be writen (only needed for write dwg 2007+)
+  //RLZ: to be written (only needed for write dwg 2007+)
```
Comment (3nids) [inline src/app/dwg/libdxfrw/intern/drw_textcodec.cpp]: this typo is already in spelling.dat, but we leave as is external libraries (libdxfrw)

File: src/app/dwg/libdxfrw/drw_entities.cpp [hunk 2]
```diff
@@ -1807,7 +1807,7 @@ bool DRW_MText::parseDwg( DRW::Version version, dwgBuffer *buf, duint32 bs )
   DRW_UNUSED( ext_ht );
   /* Extents wid BD Undocumented and not present in DXF or entget The extents
   rectangle, when rotated the same as the text, fits the actual text image on
-  the screen (altough we've seen it include an extra row of text in height). */
+  the screen (although we've seen it include an extra row of text in height). */
```
Comment (3nids) [inline src/app/dwg/libdxfrw/drw_entities.cpp]: same here

**Comment by 3nids** [comment]
As @nyalldawson pointed out, there is a file to write common misspellings. Cancellation is deliberate and the other 2 are already present in the file.
Thanks a lot anyway.

**Comment by luzpaz** [comment]
interesting..how is spelling.dat used?

**Comment by 3nids** [comment]
see https://github.com/qgis/QGIS/tree/master/scripts/spell_check
you can check_spelling.sh without argument, it will run on all source files.

apparently @jef-n wants to fix spelling in libdxf too 0d76dff305578b4c0d2289f0cf5909ef0cfa982b

**Comment by luzpaz** [comment]
awesome. thanks for the link. cheers!

**Comment by jef-n** [comment]
Yes, I already fixed a lot earlier - also astyled it etc.

**Pull request closed without merging**